tests/components/zwave_js/test_discovery_data_template.py

```diff
@@ -0,0 +1,44 @@
+"""Test discovery data templates."""
```

Comment:
I don't think we should test the templates directly. They should be tested indirectly by the platform tests for the platforms that use the templates.

Reply:
Alright, I've dropped it.